src/main/java/org/broadinstitute/consent/http/db/DarCollectionDAO.java

```diff
@@ -0,0 +1,157 @@
+   *
+   * FilterTerms filter on dar project title, datasetNames, collection dar code, collection update date, and DAC
+   * SortField can be projectTitle, datasetNames, dar code, update date, or DAC
+   * SortDirection can be ASC or DESC
+   *
+   * @return List<DarCollection>
+   */
+
+ @RegisterBeanMapper(value = DarCollection.class)
+ @RegisterBeanMapper(value = DataAccessRequest.class, prefix = "dar")
+ @UseRowReducer(DarCollectionReducer.class)
+ @SqlQuery(projectSortFilterQuery)
+ List<DarCollection> findAllDARCollectionsWithFilters(
+                                                    @Bind("institutionSearchTerm") String institutionSearchTerm,
+                                                    @Bind("projectSearchTerm") String projectSearchTerm,
+                                                    @Bind("researcherSearchTerm") String researcherSearchTerm,
+                                                    @Bind("darCodeSearchTerm") String darCodeSearchTerm,
+                                                    @Bind("datasetSearchTerm") String datasetSearchTerm,
+                                                    @Bind("offset") Integer offset,
+                                                    @Bind("limit") Integer limit,
+                                                    @Define("sortField") String sortField,
+                                                    @Define("sortOrder") String sortOrder);
+
+  /**
+   * Find the DARCollection and all of its Data Access Requests that contains the DAR with the given referenceId
+   *
+   * @return DarCollection
+   */
+  @RegisterBeanMapper(value = DarCollection.class)
+  @RegisterBeanMapper(value = DataAccessRequest.class,  prefix = "dar")
+  @UseRowReducer(DarCollectionReducer.class)
+  @SqlQuery(
+    "SELECT " + 
+      "c.*, dar.id AS dar_id, dar.reference_id AS dar_reference_id, dar.collection_id AS dar_collection_id, " +
+      "dar.draft AS dar_draft, dar.user_id AS dar_userId, dar.create_date AS dar_create_date, " +
+      "dar.sort_date AS dar_sort_date, dar.submission_date AS dar_submission_date, " +
+      "dar.update_date AS dar_update_date, (dar.data #>> '{}')::jsonb AS data " +
+    "FROM dar_collection c " +
+    "INNER JOIN data_access_request dar ON c.collection_id = dar.collection_id " +
+    "WHERE c.collection_id = (SELECT collection_id FROM data_access_request WHERE reference_id = :referenceId)")
+  DarCollection findDARCollectionByReferenceId(@Bind("referenceId") String referenceId);
+
+  /**
+   * Find the DARCollection and all of its Data Access Requests that has the given collectionId
+   *
+   * @return DarCollection
+   */
+  @RegisterBeanMapper(value = DarCollection.class)
+  @RegisterBeanMapper(value = DataAccessRequest.class, prefix = "dar")
+  @UseRowReducer(DarCollectionReducer.class)
+  @SqlQuery(
+      getCollectionAndDars +  " WHERE c.collection_id = dar.collection_id AND c.collection_id = :collectionId ")
```

Comment:
I didn't know this could work, nice find!

Reply:
In the get query, there's an inner join on `c.collection_id = dar.collection_id` so you don't need to add that again here. 
```suggestion
  @SqlQuery(
      getCollectionAndDars +  " WHERE c.collection_id = :collectionId ")
```